automataCI/services/io/fs.ps1

```diff
@@ -23,7 +23,7 @@ function FS-Append-File {
 
 
 	# perform file write
-	$null = Add-Content -Path $___target -Value $___content
+	$null = Add-Content -Path $___target -Value $___content -NoNewline
```

Comment:
This may affect other technologies' rendering. Need to horizontally implement all technologies to make sure it does not happen here.

Reply:
yeah. Will do that once after the upstream.